### PR TITLE
Remove period from mozmail var

### PR DESCRIPTION
**Pull request by codemist**

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -821,9 +821,9 @@ multi-part-onboarding-reply-headline = Reply to your emails
 
 multi-part-onboarding-premium-welcome-description-2 = No more five-mask limit: you can now generate as many custom or random masks as you need. On desktop, you can use the { -brand-name-relay } add-on to create them on the go.
 # Variables:
-#   $mozmail (string): domain used by Relay masks (.mozmail.com)
+#   $mozmail (string): domain used by Relay masks (mozmail.com)
 multi-part-onboarding-premium-get-domain-description-3 = With a custom subdomain, you can create masks without having to generate them beforehand.
```
Comment (peiying2) [inline en/app.ftl:825]: The change needs to have a new string ID even though it's almost identical to the old version. Without the new ID, the change won't trigger the locales that had localized the string. You can name it `…-description-2b' or something like this. And the old string should be removed so it doesn't get localized if no longer used.

**Review by peiying2 — COMMENTED** [review]
String ID needs to be changed to indicate the version change. Or update won't show up in Pontoon for localizers. 

https://github.com/mozilla-l10n/documentation/blob/main/src/localization/dev_best_practices.md#string-identifiers-ids

File: en/app.ftl (hunk 1 above)
Comment (codemist) [reply en/app.ftl:825]: Oops, you're right. Just added a new line.

File: en/app.ftl [hunk 2]
```diff
@@ -821,9 +821,9 @@ multi-part-onboarding-reply-headline = Reply to your emails
 
 multi-part-onboarding-premium-welcome-description-2 = No more five-mask limit: you can now generate as many custom or random masks as you need. On desktop, you can use the { -brand-name-relay } add-on to create them on the go.
 # Variables:
-#   $mozmail (string): domain used by Relay masks (.mozmail.com)
-multi-part-onboarding-premium-get-domain-description-3 = With a custom subdomain, you can create masks without having to generate them beforehand.
-    Need one to sign up for a newsletter? Just say “read@customsubdomain{ $mozmail }”
+#   $mozmail (string): domain used by Relay masks (mozmail.com)
+multi-part-onboarding-premium-get-domain-description-4 = With a custom subdomain, you can create masks without having to generate them beforehand.
```
Comment (peiying2) [inline en/app.ftl:825]: I meant to say -3b or -3-b as it is the 3rd paragraph or bullet so it doesn't look like we missed a paragraph and jump to -4. I am looking at the content before this string.

Comment (codemist) [reply en/app.ftl:825]: Oh, so do you meant you want the variable name to be something like `multi-part-onboarding-premium-get-domain-description-3b`?

Comment (peiying2) [reply en/app.ftl:825]: Yes, basically it is a replacement of an older version of the same string, either due to minor content update or bug fix.

Comment (codemist) [reply en/app.ftl]: Done!

**Review by peiying2 — APPROVED** [review]

**Pull request merged**